pandas/core/strings.py

```diff
@@ -2917,7 +3021,8 @@ def rindex(self, sub, start=0, end=None):
     5    3.0
     dtype: float64
     """)
-    len = _noarg_wrapper(len, docstring=_shared_docs['len'], dtype=int)
+    len = _noarg_wrapper(len, docstring=_shared_docs['len'],
+                         forbidden_types=None, dtype=int)
```

Comment:
I'm thinking that `len` should still be allowed for `bytes` to preserve a sort of minimal functionality compared to what is working as of v.0.23.4